terraformer_policy.tf

```diff
@@ -0,0 +1,34 @@
+# Create the IAM policy for the Terraformer EC2 server instances that
+# allows full access to create/destroy new resources in this account
+# and create/modify/destroy existing resources not created by the "VM
+# Fusion - Development" team.
+
+data "aws_iam_policy_document" "full_access_doc" {
+  provider = aws.provisionassessment
+
+  # Allow full access to new resources and existing resources that
+  # _were not_ created by the "VM Fusion - Development" team.
```

Comment:
Shouldn't this comment be written more generically, since we are comparing against the value of `var.tags["Team"]` (which may or may not be "VM Fusion - Development")?

Reply:
See commit acde207.

Reply:
That works for me, thanks! 👍